core/utility.ml

```diff
@@ -1459,4 +1459,13 @@ struct
         Lwt.return (x :: xs)
 end
 
+(** remote client->server call *)
+let is_remote_call params =
+  mem_assoc "__name" params && mem_assoc "__args" params
 
+(** return __result from server->client call with server continuation __continuation *)
+let is_client_return params =
+  mem_assoc "__continuation" params && mem_assoc "__result" params
+
+let is_ajax_call cgi_args =
```

Comment:
If you're moving this from `webif.ml`, could you make sure you delete it from there, and fix up the references to it from other files?

(Maybe this would be better in `RequestData`, actually).

Reply:
Sure